client/gutenberg/extensions/tiled-gallery/view.js

```diff
@@ -1,3 +1,8 @@
+/**
+ * External dependencies
+ */
+import '@babel/polyfill';
```

Comment:
I'd like to compare the size change for the view script with this addition.

If we depend on `wp-polyfill` we can share the cost of the dependency with any other code that may be using it.

I'd like to consider that before moving ahead with this.

Reply:
wp-polyfill (v7.0.0) 30.7 KB minified

view.js when building minified production versions:
- without polyfill 7.96 KiB
- with polyfill 87.6 KiB

So looks like we're better enqueing wp-polyfill as a emmediate fix and then later on we'll fix the builder.

Reply:
>     So looks like we're better enqueing wp-polyfill as a immediate fix

https://github.com/Automattic/jetpack/pull/11487